transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -301,8 +301,7 @@ protected int scheduleRead0(boolean first, boolean socketIsEmpty) {
             if (channelConfig.getUseIoUringBufferGroup() && IoUring.isRegisterBufferRingSupported() && first) {
```

Comment:
It's time to consider removing this `first` condition. I think that since we can  refill the buffer ing, we can use ReadProviderBuffer all the time,WDYT?

Reply:
I want to do some more testing and implement multishot + recv bundles first . But yep 👍